src/sentry/static/sentry/app/views/organizationEventsV2/data.jsx

```diff
@@ -65,36 +69,69 @@ export const ALL_VIEWS = deepFreeze([
 export const SPECIAL_FIELDS = {
   event: {
     fields: ['title', 'id', 'project.name'],
-    renderFunc: (data, org) => (
+    renderFunc: (data, {organization}) => (
       <Container>
         <Link
           css={overflowEllipsis}
-          to={`/organizations/${org.slug}/projects/${data['project.name']}/events/${
-            data.id
-          }/`}
+          to={`/organizations/${organization.slug}/projects/${
+            data['project.name']
+          }/events/${data.id}/`}
         >
           {data.title}
         </Link>
       </Container>
     ),
   },
+  project: {
+    fields: ['project.name'],
+    renderFunc: (data, {organization}) => {
+      const project = organization.projects.find(p => p.slug === data['project.name']);
+      return (
+        <Container>
+          {project ? (
+            <ProjectBadge project={project} avatarSize={16} />
+          ) : (
+            data['project.name']
+          )}
+        </Container>
+      );
+    },
+  },
   user: {
-    fields: ['user.email', 'user.ip'],
-    renderFunc: data => <Container>{data['user.email'] || data['user.ip']}</Container>,
+    fields: ['user', 'user.name', 'user.email', 'user.ip'],
+    renderFunc: (data, {onSearch}) => {
+      const userObj = {
+        name: data['user.name'],
+        email: data['user.email'],
+        ip: data['user.ip'],
+      };
```

Comment:
Should we have a util function for this?

Reply:
I don't know that we'll need this logic elsewhere - let's do that if it comes up?